Resources/views/notifications.html.twig

```diff
@@ -1,15 +1,17 @@
-{% for flash_message in app.session.flashbag.get('success') %}
```

Comment:
Even beter would be to wrap this in a block: `notifications`

Reply:
The include or the contents of the include?

Reply:
The full content of the file

Reply:
Added block in a25ff47